README.md

```diff
@@ -760,59 +695,60 @@ Chocolatey v0.9.9.9+.
 
-The value of the config setting. If the name includes "password", then the value
+(**Property**: This parameter represents a concrete state on the target system.)
+
+Specifies the value of the config setting. If the name includes "password", then the value
 is not ensurable due to being encrypted in the configuration file.
 
 
 ### Class: chocolatey
 
-Used for managing installation and configuration of Chocolatey itself.
+Manages installation and configuration of Chocolatey itself.
 
 #### Parameters
 
 ##### `choco_install_location`
 
-Where Chocolatey install should be located. This needs to be an absolute path starting with a drive letter e.g. `c:\`. Defaults to the currently detected install location based on the `ChocolateyInstall` environment variable, falls back to `'C:\ProgramData\chocolatey'`.
+Specifies where Chocolatey install should be located. Valid options: Must be an absolute path starting with a drive letter, for example: `c:\`. Default: The currently detected install location based on the `ChocolateyInstall` environment variable. If not specified, falls back to `'C:\ProgramData\chocolatey'`.
 
-**NOTE:** Puppet can install Chocolatey and configure Chocolatey/install packages during the same run *UNLESS* you specify this setting. This is due to the way the providers search for suitability of the command, falling back to the default install for the executable when none is found. Since environment variables and commands
-do not refresh during the same Puppet run (due somewhat to a Windows limitation with updating environment information for currently running processes), installing to a directory that is not the default won't be detected until the next time Puppet runs. So unless you really want this installed elsewhere and don't have a current existing install in that non-default location, the recommendation is not to set this value.
+**NOTE:** Puppet can install Chocolatey and configure Chocolatey install packages during the same run *UNLESS* you specify this setting. This is due to the way the providers search for suitability of the command, falling back to the default install for the executable when none is found. Because environment variables and commands
+do not refresh during the same Puppet run (due somewhat to a Windows limitation with updating environment information for currently running processes), installing to a directory that is not the default won't be detected until the next time Puppet runs. So unless you really want this installed elsewhere and don't have a current existing install in that non-default location, do not set this value.
 
 ##### `use_7zip`
 
-Whether to use built-in shell or allow installer to download 7zip to extract `chocolatey.nupkg` during installation. Defaults to `true`.
+Specifies whether to use the built-in shell or allow the installer to download 7zip to extract `chocolatey.nupkg` during installation. Valid options: `true`, `false`. Default: `true`.
```

Comment:
I think the default is actually false now. Looks like we missed updating the documentation here. My bad.